iothub/device/src/DeviceClient.cs

```diff
@@ -540,87 +539,64 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         /// </summary>
         /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
         /// <exception cref="OperationCanceledException">Thrown when the operation has been canceled.</exception>  
         /// <returns>The device twin object for the current device</returns>
-        public Task<Twin> GetTwinAsync(CancellationToken cancellationToken) => this.internalClient.GetTwinAsync(cancellationToken);
+        public Task<Twin> GetTwinAsync(CancellationToken cancellationToken) => _internalClient.GetTwinAsync(cancellationToken);
 
         /// <summary>
         /// Push reported property changes up to the service.
         /// </summary>
         /// <param name="reportedProperties">Reported properties to push</param>
         public Task UpdateReportedPropertiesAsync(TwinCollection reportedProperties) =>
-            this.internalClient.UpdateReportedPropertiesAsync(reportedProperties);
+            _internalClient.UpdateReportedPropertiesAsync(reportedProperties);
 
         /// <summary>
         /// Push reported property changes up to the service.
         /// </summary>
         /// <param name="reportedProperties">Reported properties to push</param>
         /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
-        /// <exception cref="OperationCanceledException">Thrown when the operation has been canceled.</exception>  
+        /// <exception cref="OperationCanceledException">Thrown when the operation has been canceled.</exception>
         public Task UpdateReportedPropertiesAsync(TwinCollection reportedProperties, CancellationToken cancellationToken) =>
-            this.internalClient.UpdateReportedPropertiesAsync(reportedProperties, cancellationToken);
-
-#region Device Streaming
-        /// <summary>
-        /// Waits for an incoming Cloud-to-Device Stream request.
-        /// </summary>
-        /// <returns>A stream request when received</returns>
-        public Task<DeviceStreamRequest> WaitForDeviceStreamRequestAsync() 
-            => this.internalClient.WaitForDeviceStreamRequestAsync();
+            _internalClient.UpdateReportedPropertiesAsync(reportedProperties, cancellationToken);
 
+        #region Device Streaming
         /// <summary>
-        /// Waits for an incoming Cloud-to-Device Stream request.
+        /// Waits for an incoming device stream request.
         /// </summary>
         /// <param name="cancellationToken">Token used for cancelling this operation.</param>
-        /// <returns>A stream request when received</returns>
-        public Task<DeviceStreamRequest> WaitForDeviceStreamRequestAsync(CancellationToken cancellationToken)
-            => this.internalClient.WaitForDeviceStreamRequestAsync(cancellationToken);
+        /// <returns>A stream request when received.</returns>
+        public Task<DeviceStreamRequest> WaitForDeviceStreamRequestAsync(CancellationToken cancellationToken = default)
```

Comment:
this is a good change for all of our current APIs!

Reply:
I'd agree. We can do that during our tech debt work, if you like.